server/src/main/java/org/elasticsearch/cluster/coordination/PreVoteCollector.java

```diff
@@ -62,7 +62,6 @@
         this.updateMaxTermSeen = updateMaxTermSeen;
         this.electionStrategy = electionStrategy;
 
-        // TODO does this need to be on the generic threadpool or can it use SAME?
```

Comment:
No, `SAME` is bad because `updateMaxTermSeen.accept` may block on `Coordinator#mutex`.